### PR TITLE
feat: stats-pagination

### PR DESCRIPTION
**Pull request by AmirSa12**

### CODE REVIEW

File: packages/backend/server/routes/stats.get.ts [hunk 1]
```diff
@@ -1,40 +1,23 @@
...
+    let cursor = query.cursor || undefined;
     let objectCount = 0;
 
     const encoder = new TextEncoder();
-
+    
     const packagesPrefix = `${usePackagesBucket.base}:`;
     const cursorsPrefix = `${useCursorsBucket.base}:`;
 
     const stream = new ReadableStream({
```
Comment (Aslemammad) [inline packages/backend/server/routes/stats.get.ts:17]: now we do not have any use for readable stream, let's remove it!

File: packages/backend/server/routes/stats.get.ts [hunk 2]
```diff
@@ -1,40 +1,23 @@
...
 export default eventHandler(async (event) => {
   try {
     const binding = useBinding(event);
 
-    let started = false
-
-    let cursor: string | undefined;
+    const query = getQuery(event);
 
+    let cursor = query.cursor || undefined;
```
Comment (Aslemammad) [inline packages/backend/server/routes/stats.get.ts:9]: how can we use the next cursor? i tried but apparently it does not work!

**Review by Aslemammad — CHANGES_REQUESTED** [review]

File: packages/backend/server/routes/stats.get.ts (hunk 2 above)
Comment (AmirSa12) [reply packages/backend/server/routes/stats.get.ts:9]: It does work

File: packages/backend/server/routes/stats.get.ts (hunk 1 above)
Comment (AmirSa12) [reply packages/backend/server/routes/stats.get.ts:17]: consider it done

**Review by Aslemammad — APPROVED** [review]
LGTM!

**Pull request merged**